docs/src/app/(docs)/react/components/switch/page.mdx

```diff
@@ -26,6 +26,38 @@ import { Switch } from '@base-ui/react/switch';
 </Switch.Root>;
 ```
 
+## Examples
+
+### Labeling a switch
+
+Label switches using an enclosing `<label>` element that wraps `<Switch.Root>`. Enclosing labels ensure that clicking on any gaps between the label and switch still toggles the control, and announce the label correctly to screen readers.
+
+The [Field](/react/components/field) component can also be used to provide accessible labels for individual switch without needing to keep track of manual `id` or `aria-label` associations. It supports both enclosing and sibling labeling patterns, along with a UX improvement to prevent double clicks from selecting the label text.
```

Comment:
```suggestion
The [Field](/react/components/field) component can also be used to provide accessible labels for individual switches, eliminating the need to track `id` or `aria-label` associations manually. It supports both enclosing and sibling labeling patterns, along with a UX improvement to prevent double clicks from selecting the label text.
```